tests/spread/core24/package-repositories/test-key-conflict/snapcraft.yaml

```diff
@@ -0,0 +1,31 @@
+name: test-key-conflict
+version: '1.0'
+summary: test package repos with keys already present on the system
+description: test package repos with keys already present on the system
+confinement: strict
+base: core24
+platforms:
+  amd64:
+
+package-repositories:
+  # In core24 this pair of url + suite is already listed in
+  # /etc/apt/sources.list.d/ubuntu.sources. Add it here to make sure this setup
+  # is working.
+  - type: apt
```

Comment:
As with my comment on craft-archives, is it worth putting an Ubuntu Ports repo in here and seeing if we can stage something from one of those platforms?

Reply:
Tiago explained separately that this wouldn't cause an issue.